src/PointwiseFunctions/AnalyticSolutions/RadiationTransport/MonteCarlo/HomogeneousSphere.cpp

```diff
@@ -0,0 +1,216 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticSolutions/RadiationTransport/MonteCarlo/HomogeneousSphere.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+namespace RadiationTransport::MonteCarlo::Solutions {
+
+namespace {
+double get_mask(const tnsr::I<double, 3>& x, const double radius_bound) {
+  const double radius = sqrt((x.get(0) * x.get(0)) + (x.get(1) * x.get(1)) +
```

Comment:
I believe we have a Euclidean magnitude function in `DataStructures/Tensor/EagerMath/Magnitude.hpp`